backend/src/services/kms/kms-service.ts

```diff
@@ -787,13 +787,16 @@ export const kmsServiceFactory = ({
             return projectDataKey;
           }
         }
+      } catch (error) {
+        logger.error(error, "Failed to get project data key");
```

Comment:
format logs like this so you can find them:

`method-name: some text [param1=1] [another-param=2]`

`getProjectSecretManagerKmsDataKey: Failed to get project data key for [projectId=${projectId}`